### PR TITLE
feat: branch migration statuses

### PR DESCRIPTION
**Pull request by alaister**
## What kind of change does this PR introduce?

feature

## What is the new behaviour?

Adds status labels for `RUNNING_MIGRATIONS`, `MIGRATIONS_FAILED`, and `MIGRATIONS_PASSED`

### CODE REVIEW

File: apps/studio/components/ui/Icons.tsx [hunk 1]
```diff
@@ -20,7 +22,7 @@ export const WarningIcon = ({ className }: { className?: string }) => {
     <svg
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 21 20"
-      className={`w-4 h-4 p-0.5 text-warning-200 bg-warning-600 rounded ${className}`}
+      className={cn('w-4 h-4 p-0.5 text-warning-200 bg-warning-600 rounded', className)}
```
Comment (MildTomato) [inline apps/studio/components/ui/Icons.tsx:25]: nice

File: apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx [hunk 2]
```diff
@@ -36,7 +53,7 @@ const BranchStatusBadge = ({ status }: BranchStatusBadgeProps) => {
 
   return (
     <Badge color={isUnhealthy ? 'red' : 'slate'} className="flex items-center gap-1.5">
-      {isUnhealthy && <WarningIcon className="w-3 h-3" />}
+      {isUnhealthy && <WarningIcon className="w-3.5 h-3.5 rounded-full bg-red-1000" />}
```
Comment (MildTomato) [inline apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx:56]: `bg-destructive` please

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx (hunk 2 above)
Comment (alaister) [reply apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx:56]: That's just for the background on the icon, <Badge> can only take red, so if I use `bg-destructive` it looks yucky:
Screenshot 2024-03-06 at 21 57 54

**Review by jordienr — APPROVED** [review]

File: apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx (hunk 2 above)
Comment (MildTomato) [reply apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx:56]: ah sorry, lemme check.

Comment (MildTomato) [reply apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx:56]: we don't ship anything without using theme colors now, so still blocked

Comment (MildTomato) [reply apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx:56]: I'll jump in PR, will adjust the Badge component as well

Comment (MildTomato) [reply apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx:56]: update - sent @alaister the codes

**Review by MildTomato — CHANGES_REQUESTED** [review]
sent you the code to use

**Review by MildTomato — APPROVED** [review]
🚢

**Pull request merged**